archinstall/lib/installer.py

```diff
@@ -325,7 +325,7 @@ def post_install_check(self, *args :str, **kwargs :str) -> List[str]:
 
 	def enable_multilib_repository(self):
 		# Set up a regular expression pattern of a commented line containing 'multilib' within []
-		pattern = re.compile("^#\\[.*multilib*\\]$")
+		pattern = re.compile("^#\s*\[multilib\]$")
```

Comment:
You'll need the `r""` syntax to be able to use single backslashes, otherwise the backslashes will need to be escaped because it's a normal python string.